content/en/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions.md

```diff
@@ -1070,17 +889,8 @@ The column's `format` controls the style used when `kubectl` prints the value.
 
 ### Subresources
 
-{{< feature-state state="stable" for_k8s_version="v1.16" >}}
-
 Custom resources support `/status` and `/scale` subresources.
 
-You can disable this feature using the `CustomResourceSubresources` feature gate on
-the [kube-apiserver](/docs/admin/kube-apiserver):
```

Comment:
same for feature gates

Reply:
This one has graduated as well, so can be removed.